src/components/App.js

```diff
@@ -1,27 +1,35 @@
-import React, { Component } from 'react';
-import axios from 'axios';
-import { BACKEND_URL } from '../config/constants';
+import React, { Fragment } from 'react';
 
-class App extends Component {
-  state = {
-    status: '',
-  }
+import { BrowserRouter as Router, Switch, Route, NavLink } from 'react-router-dom';
 
-  componentDidMount() {
-    axios.get(BACKEND_URL).then((data) => {
-      this.setState({
-        status: data.data.status,
-      });
-    });
-  }
+import routes from '../config/routes';
+
+import Home from '../components/Home';
+import Auth from '../components/Auth';
+
+const App = () => (
+  <Router>
+    <Fragment>
```

Comment:
Is there any need for a `Fragment` here?

Reply:
Yes, as <Router> takes only a single child.